config/accounts.go

```diff
@@ -19,6 +19,7 @@ type Account struct {
 	EventsEnabled bool        `mapstructure:"events_enabled" json:"events_enabled"`
 	CCPA          AccountCCPA `mapstructure:"ccpa" json:"ccpa"`
 	GDPR          AccountGDPR `mapstructure:"gdpr" json:"gdpr"`
+	DebugAllowed  bool        `mapstructure:"debug_allowed" json:"debug_allowed"`
```

Comment:
Should be DebugAllow / debug_allow per the spec and to coordinate with debug.allow in the bidder info.

Reply:
Done